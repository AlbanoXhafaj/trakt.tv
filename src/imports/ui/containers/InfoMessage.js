import React from "react";
import PropTypes from 'prop-types';
// a centered div with a text displaying an info
const InfoMessage = ({ children }) => (
  <div className="infoMessage">
    <h3>
      { children }
    </h3>
  </div>
)

InfoMessage.propTypes = {
  children: PropTypes.string.isRequired
};

export default InfoMessage;
