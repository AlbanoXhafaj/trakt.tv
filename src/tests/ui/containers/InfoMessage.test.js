import "../../setups/setup_enzyme.js";
import React from "react";

import { shallow , mount } from "enzyme";
import toJSON from 'enzyme-to-json';

import InfoMessage from "containers/InfoMessage";

describe("<InfoMessage />" ,() => {
  const message = "Message";

  const wrapper = shallow(<InfoMessage>{ message }</InfoMessage>);

  it("render without problem", () => {
    expect(wrapper.length).toBe(1);
  })

  it("should match the snapshot", () => {
    expect(toJSON(wrapper)).toMatchSnapshot();
  })
});
