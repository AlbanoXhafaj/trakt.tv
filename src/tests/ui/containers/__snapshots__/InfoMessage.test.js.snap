// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<InfoMessage /> should match the snapshot 1`] = `
<div
  className="infoMessage"
>
  <h3>
    Message
  </h3>
</div>
`;
